### PR TITLE
INTG-3044 courses feed

### PR DESCRIPTION
**Pull request by MickStanciu**
## Summary

Add feed course training progress

## Checklist

- [x] Is this PR a reasonable size?

- [ ] List deployment sequence with all relevant PRs.
  - ...

---

**Code Review Guidelines** for Reviewers

- Try to review in a timely manner. Opinions/nitpicks should not be blockers. Pair on a call for non-trivial feedback.
- Overall design and approach should follow established patterns. Don't try to make the PR perfect.
- Try to identify edge cases, race conditions, over-engineering, lack of test coverage and complexity.
- If you don't feel qualified to review the code, pass it on to someone who is.

### CODE REVIEW

File: .github/workflows/release-connector.yml [hunk 1]
```diff
@@ -58,6 +58,6 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       with:
         upload_url: ${{ steps.create_release.outputs.upload_url }}
-        asset_path: ./Sample Report.pbit
+        asset_path: ./Power BI Templates/General Report/SafetyCulture Report Template.pbit
```
Comment (HarryTranAU) [inline .github/workflows/release-connector.yml:61]: Is the SC report template the same as the sample report?

Comment (MickStanciu) [reply .github/workflows/release-connector.yml:61]: So this is a bug fix. A previous commit renamed/moved the template in another folder and forgot to update this file

**Review by HarryTranAU — APPROVED** [review]
